easybuild/easyconfigs/p/python-irodsclient/python-irodsclient-1.1.4-GCCcore-11.2.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'PythonBundle'
+
+name = 'python-irodsclient'
+version = '1.1.4'
+
+homepage = 'https://github.com/irods/python-irodsclient'
+description = "A python API for iRODS"
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
```

@jefscheepers please add `binutils` builddep here?
```suggestion

builddependencies = [('binutils', '2.37')]

```